app/controllers/agenda/minutes.js

```diff
@@ -114,11 +113,12 @@ function getAttendees(model) {
     .map(mandateeName)
     .filter((x) => !seen.has(x));
 
+  const secretary = mandateeName(model.secretary);
```

Comment:
This should probably deal with a secretary that's null/undefined (for existing agenda's where no secretary is set).

```js
let secretaryName = '';
const secretary = /*fetch secretary here*/;
if (secretary) {
  secretaryName = mandateeName(secretary);
}

// ....

return {
  primeMinister,
  viceMinisters,
  ministers,
  secretary: secretaryName,
};